packages/ppx_relit/ppx_relit.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "ppx_relit"
+version: "0.1"
+maintainer: "Charles Chamberlain <charlespipin@gmail.com>"
+authors: [
+  "Charles Chamberlain <charlespipin@gmail.com"
+  "Cyrus Omar <cyrus.omar@gmail.com"
+]
+homepage: "https://github.com/cyrus-/relit"
+bug-reports: "https://github.com/cyrus-/relit/issues"
+license: "MIT"
+dev-repo: "https://github.com/cyrus-/relit"
+build: [
+  ["jbuilder" "build"]
+]
+install: ["jbuilder" "install" "ppx_relit"]
+remove: ["ocamlfind" "remove" "project"]
+depends: "ocamlfind"
```

Comment:
This depends at least on jbuilder, but looking at the jbuild file there are a lot of dependencies missing: 
https://github.com/cyrus-/relit/blob/master/ppx_relit/jbuild